### PR TITLE
Add english equivalents. Add function from new versions

### PR DESCRIPTION
**Pull request by Kirill**
### Changes
Add functions from new versions of platform.
Add english equivalents for each russian names.

### CODE REVIEW

File: src/languages/1c.js [hunk 1]
```diff
@@ -513,6 +1819,10 @@ export default function(hljs) {
 
   return {
     name: '1C:Enterprise',
+    aliases: [
+      '1c',
```
Comment (joshgoebel) [inline src/languages/1c.js:1823]: Unnecessary.

**Comment by joshgoebel** [comment]
Thanks might take me a little while before I can dig into this one (to look at the autodetect stuff), lot going on right now.

**Comment by Kirill** [comment]
I need something or wait?

**Comment by joshgoebel** [comment]
Nope, ball is in my court.

**Comment by joshgoebel** [comment]
> Add english equivalents for each russian names.

1C allows you to program in English and Russian?

File: src/languages/1c.js [hunk 2]
```diff
@@ -1819,10 +1819,6 @@ export default function(hljs) {
 
   return {
     name: '1C:Enterprise',
-    aliases: [
-      '1c',
-      'bsl'
```
Comment (joshgoebel) [inline src/languages/1c.js]: How did we conclude this is no longer an alias?

**Comment by Kirill** [comment]
> > Add english equivalents for each russian names.
> 
> 1C allows you to program in English and Russian?

Yes. Each naming have dual equivalents.

File: src/languages/1c.js (hunk 2 above)
Comment (Kirill) [reply src/languages/1c.js]: I don't remember.

Comment (Kirill) [reply src/languages/1c.js]: I must send new commit with rollback this change?

Comment (joshgoebel) [reply src/languages/1c.js]: Yes, please.

Comment (Kirill) [reply src/languages/1c.js]: Return aliases

**Review by ffontecha080 — APPROVED** [review]
One of the best

**Comment by joshgoebel** [comment]
1C is leaving the core library (with the next release).  If you'd like to contribute to the new 3rd party grammar that I've just now extracted you can find it at: https://github.com/highlightjs/highlightjs-1c-enterprise/

If you want to re-open this PR there I think we can get it merged, but we'll need to find a long term maintainer for the grammar. 

_Closing this as no further PRs against 1C will be accepted to core._

**Pull request closed without merging**